Ports/harfbuzz/package.sh

```diff
@@ -5,17 +5,22 @@ files="https://github.com/harfbuzz/harfbuzz/releases/download/${version}/harfbuz
 useconfigure='true'
 auth_type='sha256'
 depends=("freetype" "libicu")
-configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DHB_HAVE_FREETYPE=ON" "-DHB_HAVE_ICU=ON")
+configopts=(
+	"-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+	"-DHB_HAVE_FREETYPE=ON"
+	"-DHB_HAVE_ICU=ON"
+	"-DCMAKE_C_FLAGS=-lfreetype"
+	"-Bbuild"
```

Comment:
If you're reformatting these, please also make them use single quotes instead of double quotes where no string interpolation takes place.

Reply:
And spaces rather than tabs 🚫 📑